packages/broadcast/src/browser/broadcast-frontend-module.ts

```diff
@@ -0,0 +1,29 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
Are you working at TypeFox? We surely aren't in 2018 :)

Reply:
Nope 😛 Tslint asks for file header. It is okay to put my name, or what should I put?

Reply:
You should put who ever owns the copyright of your work. :-) 
So if you are in doubt, you should add your name, yes.